javascript/ql/src/Security/CWE-347/MissingJWTKeyVerification.ql

```diff
@@ -10,12 +10,11 @@
  */
 
 import javascript
-import DataFlow
 import semmle.javascript.RestrictedLocations
 
-from CallNode call
+from DataFlow::CallNode call
 where
-  call = moduleMember("jsonwebtoken", "verify").getACall() and
+  call = DataFlow::moduleMember("jsonwebtoken", "verify").getACall() and
   unique(boolean b | b = call.getArgument(1).analyze().getABooleanValue()) = false
 select call.asExpr().(FirstLineOf),
```

Comment:
Could we select the faulty argument instead with a message like:

`This argument disables the integrity enforcement of the token verification.`